master/usage/configuration/mtu.md

```diff
@@ -83,6 +84,9 @@ Example CNI configuration
 }
 ```
 
+By default, CNI configuration mtu is set to 1500 by virtue of `"mtu": __CNI_MTU__` variable.
```

Comment:
This isn't quite right - the `__CNI_MTU__` variable only affects the CNI plugin. However, the "mtu" option in the ConfigMap is passed to both the CNI plugin and Felix as well.

Reply:
@caseydavenport thanks for your time for review. I have pushed the change taking care of these comments.